### ISSUE TITLE
URL for API Documentation gives no result

### ISSUE THREAD

**Issue by jmp3833** [Pastafarians/linguine#22]
Looks like a URL was created to provide API documentation but was either never finished or is not registered correctly in the application. 

Steps to reproduce: 
Either on linguine.rit.edu or a local build open the developer console of your favorite web browser to find this error message: 

<img width="734" alt="screen shot 2015-09-29 at 4 52 04 pm" src="https://cloud.githubusercontent.com/assets/1236565/10177941/e45d0730-66ca-11e5-81c0-dd0f0ab8e0de.png">

Proposed resolution: 
Edit the Angular module and controllers to display the existing page and identify what documentation already exists.

**Comment by daniellegonzalez** [comment]
edit to display initial page - content in different story

**Comment by daniellegonzalez** [comment]
I already investigated the documentation branch left behind from the previous team. I could not find any differences between this and the current branch in regards to this bug. I made some modifications to the source code because the reference to the documentation module was not being loaded into the index. 

PLAN OF ACTION
1. talk to Kristen to see if the knows what the issue is
2. remove the doc module and re-add a new one
